src/web/components/SharingPermission/SharingPermissionsTable.scss

```diff
@@ -0,0 +1,85 @@
+.sharing-permissions-table-header-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 1rem;
+}
+
+.sharing-permissions-table h1 {
+  font-weight: 600;
+  font-size: 1.125rem;
+  line-height: 1.375rem;
+}
+
+.sharing-permissions-search-bar-container {
+  width: 190px;
+  display: flex;
+  align-items: center;
+  border-bottom: 2px solid var(--theme-search-border);
+  padding: 6px 2px;
+  justify-content: space-between;
+  ::placeholder {
+    /* Chrome, Firefox, Opera, Safari 10.1+ */
+    color: var(--theme-search-border);
+    opacity: 1; /* Firefox */
+  }
+
+  :-ms-input-placeholder {
```

Comment:
Why the vendor-specific versions? Caniuse.com suggests this has been in everything for years now.

Reply:
I will test in different browser and check if we could remove them

Reply:
@lionell-pack-ttd I've tested it in edge and it works without this.